### DISCUSSION TITLE
Can you add some features?

### DISCUSSION THREAD

**Discussion by kunedo** [tanujnotes/Olauncher#141] [Ideas]
Hi @tanujnotes

I like your launcher. But can you add some features? 

1. **Disable open the app automatically.** At the first, I thought it's bug. After using it a few times, I found out that the app opened automatically. I prefer to open the app manually.

2. **Add apps to home screen directly.** Now, It's kind of troublesome. I have to set the number of apps on the home screen before I can add apps to the home screen.

3. **Add slogan on the home screen.** I like to add a slogan, like keep it simple, on the home screen to remind me. 

And some features that I think are less needed:

1. **Alphabet on the right of the app drawer.** Searching is good, but I am used to find apps by alphabet. And it would be better if support Chinese.

2. **Set the position of date/time and apps blocks on the home screen.** When hiding the date/time, there is a lot of blank space at the top of apps. Just set the top margin of blocks would be fine. 

3. **Set the alignment of the blocks individually.** Like set the date/time to align to the left, and the apps to the right.

Sorry for my poor English.

Have a nice day.

**Comment by nodiscardme** [comment]
About 1, I like this feature. It opens as soon as I type first few letters.

**Reply by Ilithy** [reply]
Same, I find the point 1 very practical, and offers flawless responsiveness, but having the possibility of disabling it is not a bad idea.

For points 2 and 3, personally, I don't find it useful, I like Olauncher for its sobriety and its minimalism.

For the additional points number 3 _Set the alignment of the blocks individually._, I'm really for it, it's a simple customization but which would fill the screen more fairly (in my opinion)
thank you.

**Reply by OndrejKulhavy** [reply]
The way it opens directly the app I'm searching for is one of the main reasons why I love this luncher so much.
